packages/trader/src/Modules/CFD/Containers/cfd-dashboard.tsx

```diff
@@ -39,13 +56,107 @@ const LoadTab = ({ children, is_loading, loading_component, ...props }) => {
     return <Tabs {...props}>{children}</Tabs>;
 };
 
-const CFDDashboard = props => {
-    const [is_demo_enabled, setIsDemoEnabled] = React.useState(false);
-    const [is_real_enabled, setIsRealEnabled] = React.useState(false);
-    const [active_index, setActiveIndex] = React.useState(0);
-    const [is_demo_tab, setIsDemoTab] = React.useState(true);
-    const [is_notification_loaded, setIsNotificationLoaded] = React.useState(false);
-    const [password_manager, setPasswordManager] = React.useState({
+type TOpenAccountTransferMeta = {
+    category: string;
+    type?: string;
+};
+
+type TStandPoint = {
+    financial_company: string;
+    gaming_company: string;
+    iom: boolean;
+    malta: boolean;
+    maltainvest: boolean;
+    svg: boolean;
+};
+
+type TCFDDashboard = {
+    account_settings: { residence: string };
+    account_status: object;
+    beginRealSignupForMt5: () => void;
+    country: string;
+    createCFDAccount: () => void;
```

Comment:
`createCFDAccount` method in cfd-store gets ` ({ category, type, set_password })` args

Reply:
I've made the changes according to your suggestion.